### ISSUE TITLE
rsync gaussian output likely to crash

### ISSUE THREAD

**Issue by goord** [blue-action/ece-postprocess#11]
There is a mistake in deriving the date of the gridpoint files. Line 561 of AMET_land_surface.py reads
```python
file_time = [a for a in [a.lstrip('ICMGG{}+'.format(expname)) for a in filenames] if int(a)][0]
```
However, a.lstrip(b) means 'stripping all characters in a that occur in b, starting from the left'. So if I decide to name my experiment 'ph02' (which I did), you will get
```
In [2]: "ICMGGph02+200210".lstrip("ICMGG{}+".format("ph02"))
Out[2]: '10'
```
which is clearly not intended. The solution is obviously just to take ```int(a[-6:])```.

**Comment by goord** [comment]
Looking at the code, I am also wondering why this rsync is acually there, since cdo doesn't modify its input...

**Comment by rvanharen** [comment]
- extracting correct time fixed by 116838d
- don't rsync gaussian output fixed by fd0310
